### DISCUSSION TITLE
Charting very different values

### DISCUSSION THREAD

**Discussion by jmheist** [RomRider/apexcharts-card#185] [Q&A]
Creating a chart to show a bunch of crypto rates... with ranges from half pennies to bitcoin status, i just get flat linees since the chart has to cover so much variance. Anyway to plot them on the same graph in a more relative way?

**Comment by RomRider** [comment]
It's not the same scale so unless you use multiple y-axis, I don't see how you can plot them.
Multiple y-axis is supported, see the documentation
